.github/workflows/tests.yml

```diff
@@ -122,3 +122,12 @@ jobs:
         components: rust-docs
 
     - run: cargo +nightly-2024-04-17 doc --no-deps --workspace --document-private-items
+
+  test-release:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - uses: actions-rs/cargo@ae10961054e4aa8b4aa7dffede299aaf087aa33b # v1.0.3
```

Comment:
nit: Do we necessarily need `actions-rs`? git-hub runners already ship with cargo [installed](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#rust-tools) 😃

Reply:
Done